src/util/common.js

```diff
@@ -50,7 +52,7 @@ exports.inspect = curry((opt, a) => util.inspect(a, opt));
  *
  * @return {Object} The input.
  */
-exports.dump = tap(compose(util.log, add(__, '\n'), a => JSON.stringify(a, null, 2)));
+exports.dump = tap(compose(log.info, add(__, '\n'), a => JSON.stringify(a, null, 2)));
```

Comment:
Actually; this whole function is never used. I removed it in my `embrace-sanctuary` branch. But you might as well remove it from here.